### DISCUSSION TITLE
Metric Aggregation and Spike Aggregation documentation does not mention percentile_range and percentiles

### DISCUSSION THREAD

**Discussion by nsano-rururu** [jertel/elastalert2#741] [Ideas]
`percentile_range` and `percentiles` is a setting added in the following pull request
https://github.com/jertel/elastalert2/pull/41
https://elastalert2.readthedocs.io/en/latest/ruletypes.html#spike-aggregation
https://elastalert2.readthedocs.io/en/latest/ruletypes.html#metric-aggregation

**Comment by jertel** [comment]
I agree, it should be added. I can add it if you haven't already started on it.

**Reply by nsano-rururu** [reply]
>I agree, it should be added. I can add it if you haven't already started on it.

I haven't done anything yet. If possible, please.

**Reply by jertel** [reply]
Pushed: https://github.com/jertel/elastalert2/commit/e682ea8113bf9f413b6339e6803b5262881f2b30
